queries.sql

```diff
@@ -18,6 +18,41 @@ SELECT * from animals WHERE name <> 'Gabumon';
 
 SELECT * from animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
 
+Begin;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+Begin;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+commit;
+
+Begin;
+DELETE FROM animals; 
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=then%20roll%20back%20the%20transaction), Please rollback the transaction.

Reply:
- [ ] Again here kindly add the rollback statement so as to undo the changes made to the database as requested by the previous reviewer